sample/redo/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "redo",
+  "version": "0.1.0",
```

Comment:
Out of curiosity, what is the philosophy of versioning top-level applications in the node ecosystem? Since they don't have external APIs, does that mean they just never advance the major (or minor??) version?

Reply:
This was generated...

I don't think it matters for apps... Maybe some tools still want it?